src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let discount = 0;
+
+  if (days < 3) {
+    discount = 0;
+  }
```

Comment:
you already have a default value so there is no need to set it
```suggestion
  let discount = 0;
```